src/Vision.php

```diff
@@ -18,6 +18,13 @@ class Vision
     protected $features = [];
 
     /**
+     * @var VisionRequest
+     */
+    protected $visionRequest;
+
+    const RETURN_TYPE_JSON = 'JSON';
```

Comment:
We now actually have 2 response types. 
 - Json
 - Object

Can you move this constant to the top of the class, and rename it to RESPONSE_TYPE_ as that is what you use in your variable name.